lib/puppet-lint/plugins/check_unsafe_interpolations.rb

```diff
@@ -0,0 +1,46 @@
+PuppetLint.new_check(:check_unsafe_interpolations) do
+  COMMANDS = Set['command', 'onlyif', 'unless']
```

Comment:
Small nit and it doesn't have to change. I don't think this being a `Set` has any benefit.